nautobot/core/api/views.py

```diff
@@ -683,18 +689,15 @@ def init_graphql(self):
         if not self.schema:
             self.schema = graphene_settings.SCHEMA
 
-        if self.backend is None:
-            self.backend = get_default_backend()
-
-        self.graphql_schema = self.graphql_schema or self.schema
+        self.graphql_schema = self.graphql_schema or self.schema.graphql_schema
```

Comment:
I don't think we actually are using `self.graphql_schema` anywhere except locally within this function, I'll see if we can remove it.